addons/purchase_stock/models/stock.py

```diff
@@ -257,6 +257,21 @@ def _set_default_route_id(self):
             orderpoint_wh_supplier.route_id = route_id[0].id
         return super()._set_default_route_id()
 
+    def _get_min_qty_to_order(self):
+        """
+        Calculates the minimum quantity that can be ordered according to the PO UoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_to_order()
+        if min_qty_order_possible:
+            return min_qty_order_possible
+        elif self.route_id:
+            rule_action = self.route_id.rule_ids[0].action
+            if rule_action == "buy":
+                order_uom = self.product_id.uom_po_id
+                min_qty_order_possible = order_uom._compute_quantity(1, self.product_uom)
```

Comment:
At this point, it could be a good idea to check the supplierinfo to have a minimal amount that a supplier agree to deliver